django-stubs/utils/csp.pyi

```diff
@@ -0,0 +1,24 @@
+import sys
+
+if sys.version_info >= (3, 11):
+    from enum import StrEnum
+else:
+    from enum import Enum
+
+    class ReprEnum(Enum): ...  # type: ignore[misc]
+    class StrEnum(str, ReprEnum): ...  # type: ignore[misc]
+
+class CSP(StrEnum):
```

Comment:
Use underscore so that these aren't visible to users.

```suggestion
    from enum import StrEnum as _StrEnum
else:
    from enum import Enum

    class _ReprEnum(Enum): ...  # type: ignore[misc]
    class _StrEnum(str, _ReprEnum): ...  # type: ignore[misc]

class CSP(_StrEnum):
```